### PR TITLE
Added MsgMeasurementState

### PR DESCRIPTION
**Pull request by mbavaro**
Is this all I have to do? Or is there a libsbp pointer I should update?

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018042018-4-g891a2b0
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=502749)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018042018-4-g891a2b0&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018042018-4-g891a2b0&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018042018-4-g891a2b0
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018042018-4-g891a2b0)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018042018-4-g891a2b0/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018042018-4-g891a2b0

**Comment by silverjam** [comment]
@mbavaro If the message is coming from firmware, then we don't need to update libsbp

File: package/piksi_system_daemon/piksi_system_daemon/src/whitelists.c [hunk 1]
```diff
@@ -128,6 +129,7 @@ static port_whitelist_config_t port_whitelist_config[PORT_MAX] = {
         MsgBasePosECEF                72
         MsgObs                        74
         MsgSpecan                     81
+		MsgMeasurementState           97
```
Comment (silverjam) [inline package/piksi_system_daemon/piksi_system_daemon/src/whitelists.c]: Taaaaabs

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018042018-5-g6bbe6c6
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=502866)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018042018-5-g6bbe6c6&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018042018-5-g6bbe6c6&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018042018-5-g6bbe6c6
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018042018-5-g6bbe6c6)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018042018-5-g6bbe6c6/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018042018-5-g6bbe6c6

**Comment by mbavaro** [comment]
ah, who uses tabs anymore these days? in the process of changing development environment, I must have forgotten to disable them explicitly.. sorry!
message does come from FW, so merging early next week

**Comment by silverjam** [comment]
@mbavaro you, apparently? 😛

**Comment by mbavaro** [comment]
apparently 🙄 ..

**Pull request merged**